Scripts/script-StixCreator.yml

```diff
@@ -0,0 +1,118 @@
+          stix_type_and_value = "[file:hashes.md5 = '" + value + "']"
+      elif demisto_indicator_type.lower() == "File SHA-1".lower():
+          stix_type_and_value = "[file:hashes.sha1 = '" + value + "']"
+      elif demisto_indicator_type.lower() == "File SHA1".lower():
+          stix_type_and_value = "[file:hashes.sha1 = '" + value + "']"
+      elif demisto_indicator_type.lower() == "File SHA256".lower():
+          stix_type_and_value = "[file:hashes.sha256 = '" + value + "']"
+      elif demisto_indicator_type.lower() == "File SHA-256".lower():
+          stix_type_and_value = "[file:hashes.sha256 = '" + value + "']"
+      elif demisto_indicator_type.lower() == "IP".lower():
+          stix_type_and_value = "[ipv4-addr:value = '" + value + "']"
+      elif demisto_indicator_type.lower() == "URL".lower():
+          stix_type_and_value = "[url:value = '" + value + "']"
+      else:
+          stix_type_and_value = "[{}:value = '{}']".format(demisto_indicator_type.lower(),value)
+
+      label_as_type = demisto_indicator_type.lower()
+
+      if demisto_score.lower() == "bad":
+          demisto_score = "High"
+      elif demisto_score.lower() == "suspicious":
+          demisto_score = "Medium"
+      else:
+          demisto_score = ""
+      indicator = Indicator(labels=[label_as_type],
+                            pattern=stix_type_and_value,
+                            source=source_system,
+                            created=demisto_created,
+                            modified=last_seen,
+                            firstSeen=first_seen,
+                            score=demisto_score,
+                            allow_custom=True)
+
+      indicators.append(indicator)
+  if counter > 1:
+      bundle = Bundle(indicators)
+      context = {
+          'StixExportedIndicators(val.pattern && val.pattern == obj.pattern)': json.loads(str(bundle))
+      }
+      demisto.results({'Type': entryTypes['note'],
+                       'Contents': str(bundle),
+                       'EntryContext': context,
+                       'ContentsFormat': formats['json']})
+  else:
+
+      context = {
+          'StixExportedIndicators(val.pattern && val.pattern == obj.pattern)': json.loads(str(indicators[0]))
+      }
+
+      demisto.results({'Type': entryTypes['note'],
+                       'Contents': str(indicators[0]),
+                       'EntryContext': context,
+                       'ContentsFormat': formats['json']})
+type: python
+tags: []
+comment: Get list of indicators in argument 'indicators' and create out of them a
+  json file in STIX 2.0 format
+enabled: true
+args:
+- name: indicators
+  description: json object of all indicators  fields index mapped to indicators fields
+outputs:
+- contextPath: StixExportedIndicators.created
+  description: Indicator creation date
+  type: string
+- contextPath: StixExportedIndicators.firstSeen
+  description: Indicator first appearance
```

Comment:
add the firstSeen date format, same for created date format

Reply:
fixed